src/engine/renderer/gl_shader.h

```diff
@@ -392,11 +405,12 @@ class GLShaderManager
 	                                     Str::StringRef compileMacros, int deformIndex );
 	std::string ShaderPostProcess( GLShader *shader, const std::string& shaderText );
 	std::string BuildDeformShaderText( const std::string& steps );
-	std::string BuildGPUShaderText( Str::StringRef mainShader, GLenum shaderType ) const;
+	std::string ProcessInsert( const std::string& shaderText, const uint32_t offset ) const;
```

Comment:
Would be better named ProcessInserts (with an s)

Reply:
Changed to `ProcessInserts()`.